src/components/Player/ItemModal.vue

```diff
@@ -251,10 +252,18 @@ export default {
       // whether the type of the question is subjective if item is question
       return this.questionType == "subjective";
     },
+    onEditorPage() {
+      return this.$route.name == "Editor";
+    },
   },
   methods: {
     checkScreenOrientation() {
       // check if the device is in portrait or landscape mode
+      if (this.onEditorPage) {
+        // device is assumed to be always in landscape mode when the user is on the Editor page
+        this.isPortrait = false;
+        return;
+      }
```

Comment:
I think this be passed as param no? like can just pass isPortrait as false from editor?